<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>774</width>
    <height>576</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QPlainTextEdit" name="plainTextEdit">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>20</y>
      <width>591</width>
      <height>31</height>
     </rect>
    </property>
    <property name="placeholderText">
     <string>Image Address</string>
    </property>
   </widget>
   <widget class="QGraphicsView" name="graphicsView">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>70</y>
      <width>591</width>
      <height>441</height>
     </rect>
    </property>
   </widget>
   <widget class="QTextBrowser" name="resultsOutput">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>640</x>
      <y>70</y>
      <width>121</width>
      <height>441</height>
     </rect>
    </property>
    <property name="placeholderText">
     <string>Output</string>
    </property>
   </widget>
   <widget class="QComboBox" name="toDoList">
    <property name="geometry">
     <rect>
      <x>640</x>
      <y>20</y>
      <width>121</width>
      <height>31</height>
     </rect>
    </property>
    <item>
     <property name="text">
      <string/>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Load Image</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Gray Image and Calculate Mean</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Average Brightness</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Detect Edge</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>BlurBox</string>
     </property>
    </item>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>774</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
